CHANGELOG.rst

```diff
@@ -1,8 +1,17 @@
 Changelog
 =========
 
+0.0.1.dev8 (2018-06-04)
```

Comment:
Incorrect date

Reply:
Oh, d'oh! Somehow that's what I came up with after looking at the calendar... will fix!